puppetry/kinectv2/README.md

```diff
@@ -0,0 +1,30 @@
+An example of using the Kinectv2 sensor to capture live mocap puppetry animations
+
+PREREQUISITES:
+    Second Life Viewer supporting the most recent puppetry features and protocol format
+    Kinect v2 sdk from https://www.microsoft.com/en-us/download/details.aspx?id=44561
+    Install the https://github.com/Kinect/PyKinect2 python library latest code
+    Install pygame: https://www.pygame.org/wiki/GettingStarted
```

Comment:
I'd recommend using tk, which is available by default with python, instead of pygame if all it is being used for is UI. PyGame is a rather heavy dependency.

Reply:
I'm using pygame because the full-body capture demo from the Kinect SDK uses it.   I used that as the basis for the plug-in, since it already had image and bone capture code, and the display to draw the skeleton.   I agree pygame is heavy but that was definitely the fastest way to get something running and connected to the viewer

Reply:
Gotcha, thanks for the details.